docs/ReleaseNotes.md

```diff
@@ -48,7 +48,7 @@ Constructors of the `RecordQueryUnionPlan` and `RecordQueryIntersectionPlan` hav
 
 * **Bug fix** Union fields for record types are now assigned based on field number rather than declaration order [(Issue #620)](https://github.com/FoundationDB/fdb-record-layer/issues/620)
 * **Feature** The `MetaDataEvolutionValidator` now permits record types to be renamed as long as all occurences of the record type in the meta-data definition are also renamed [(Issue #508)](https://github.com/FoundationDB/fdb-record-layer/issues/508)
-
+* **Bug fix** The `SizeStatisticsCursorContinuation`must not access the continuation of its underlying cursor after it is exhausted [(Issue #656)](https://github.com/FoundationDB/fdb-record-layer/pull/657)
```

Comment:
This should be up above in the commented out section. Something like line 19 or so.

Reply:
Also, a space is needed before "must"